pkg/addons/translate.go

```diff
@@ -382,6 +383,14 @@ var Translations = map[string][]struct {
 			Path:     jsonpath.MustCompile("$.spec.template.spec.containers[1].env[?(@.name='MONITORING_GCS_ACCOUNT')].value"),
 			Template: "{{ .Config.GenevaLoggingControlPlaneAccount }}",
 		},
+		{
+			Path:     jsonpath.MustCompile("$.spec.template.spec.containers[1].env[?(@.name='MONITORING_GCS_ENVIRONMENT')].value"),
+			Template: "{{ .Config.GenevaLoggingControlPlaneEnvironment }}",
+		},
+		{
+			Path:     jsonpath.MustCompile("$.spec.template.spec.containers[1].env[?(@.name='MONITORING_GCS_REGION')].value"),
```

Comment:
@sozercan What is the reason behind this part of the code? https://github.com/Azure/moffet/blob/master/logging/images/mdsd/bootstrap.sh#L17-L18

Can it be just a parameter passed from RP?

Reply:
It can be passed from RP, checking imds makes less variables to be passed in

Reply:
with current pluginConfig we either way will be configuring most of the things so +1 will not make huge diff :)